compiler/lib/stdlib.ml

```diff
@@ -447,8 +447,7 @@ end = struct
 
   let shift_right = shift_op Int32.shift_right
 
-  let shift_right_logical a b =
-    shift_op Int32.shift_right_logical (Int32.logand a 0x7fffffffl) b
+  let shift_right_logical a b = shift_op Int32.shift_right_logical a b
```

Comment:
https://github.com/ocsigen/js_of_ocaml/pull/1655#discussion_r1751937120 explains the logand. Are you sure your tests are correct ?

Reply:
You’re right, my test was wrong for value −1 for instance. I believe my test is more correct now.